bundles/org.openhab.binding.spotify/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="spotify:player">
+		<instruction-set targetVersion="1">
```

Comment:
It looks like the `thingTypeVersion` property is missing for the thing type, so I don't think these instructions are actually performed? Can you try to add this, and verify that the update is working correctly?

Reply:
You're right, upgrade works - which is what I tested ofc - but I fixed it in #18749

Reply:
Are you saying it already worked without the property?

Reply:
upgrading did work, yes, which also set the thingTypeVersion acc. to the `instructions.xml`. What would probably be an issue though, is with further updates when creating a new thing without proper version.